src/Collection.php

```diff
@@ -241,6 +241,16 @@ public function findOne($condition = [], $fields = [], $options = [])
         return empty($rows) ? null : current($rows);
     }
 
+    /**
+     * Returns if a document exists
+     * @param array $condition  query condition
+     * @return bool
```

Comment:
Need `@since 2.0.39` That's new method, right?

Reply:
jep